src/lib/sql/sql_pipeline.cpp

```diff
@@ -254,7 +257,7 @@ std::string SQLPipelineMetrics::to_string() const {
 
   std::ostringstream info_string;
   info_string << "Execution info: [";
-  info_string << "PARSE: " << parse_time_micros.count() << " µs, ";
+  info_string << "PARSE: " << parse_time_nanos.count() << " µs, ";
```

Comment:
Did you forget to change the unit? `us` to `ns`?

Reply:
Thinking about it: we would mix `us` and `ns`, could that be confusing?

Reply:
Changing it to format_duration should solve that.